script/env/dolphinscheduler_env.sh

```diff
@@ -33,3 +33,10 @@ export SEATUNNEL_HOME=${SEATUNNEL_HOME:-/opt/soft/seatunnel}
 export CHUNJUN_HOME=${CHUNJUN_HOME:-/opt/soft/chunjun}
 
 export PATH=$HADOOP_HOME/bin:$SPARK_HOME/bin:$PYTHON_HOME/bin:$JAVA_HOME/bin:$HIVE_HOME/bin:$FLINK_HOME/bin:$DATAX_HOME/bin:$SEATUNNEL_HOME/bin:$CHUNJUN_HOME/bin:$PATH
+
+# applicationId auto collection related configuration
+export HADOOP_CLASSPATH=`hadoop classpath`:${DOLPHINSCHEDULER_HOME}/tools/libs/*
+export SPARK_DIST_CLASSPATH=$HADOOP_CLASSPATH:$SPARK_DIST_CLASS_PATH
+export HADOOP_CLIENT_OPTS="-javaagent:${DOLPHINSCHEDULER_HOME}/tools/libs/aspectjweaver-1.9.7.jar":$HADOOP_CLIENT_OPTS
+export SPARK_SUBMIT_OPTS="-javaagent:${DOLPHINSCHEDULER_HOME}/tools/libs/aspectjweaver-1.9.7.jar":$SPARK_SUBMIT_OPTS
+export FLINK_ENV_JAVA_OPTS="-javaagent:${DOLPHINSCHEDULER_HOME}/tools/libs/aspectjweaver-1.9.7.jar":$FLINK_ENV_JAVA_OPTS
```

Comment:
It looks weird :thinking:  , do we have to add this OPTS here?

Reply:
@caishunfeng 
These extra java opts are necessary, do you have any suggestions to place these configuration elsewhere?

Reply:
Maybe add some annotation or make statement in docs that only configure that collect appId in aop way need these opts, WDYT?

Reply:
@caishunfeng Please have a look at the latest change, include most of your suggestions

Reply:
@ruanwenjun Any suggestions will be appreciated! It seems that everything is okay except for this part.

Reply:
@ruanwenjun This PR has to be merged in this month. Please have a look, thanks!